vnext/template/shared-lib/proj/ExperimentalFeatures.props

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <AppProjectExperimentalFeaturesProps Condition="'$(AppProjectExperimentalFeaturesProps)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'windows\ExperimentalFeatures.props'))\windows\ExperimentalFeatures.props</AppProjectExperimentalFeaturesProps>
-  </PropertyGroup>
-  <Import Project="$(AppProjectExperimentalFeaturesProps)" Condition="Exists('$(AppProjectExperimentalFeaturesProps)')"/>
+
+  <!--
```

Comment:
meta (not needed in this PR): We should see about renaming this file again since it's more like SolutionSettings or something like that

Reply:
I can give some context on why I moved it from a generic "BuildFlags" to "ExperimentalFeatures". Some of it was to promote coverage of new stuff we are pushing out, but also I recognized the user-facing property sheet was a footgun.

Props there can effect MSRN DLL compilation per flag in a way we cannot replicate with binary distribution. Make it too generic, and we will have take backs.